### PR TITLE
Update Tenant Operator Documentation

### PR DESCRIPTION
**Pull request by abdulhaseeb2**

### CODE REVIEW

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-c15538f6`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-6f954695`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-593c24ca`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-c164c69f`

File: .vuepress/config.js [hunk 1]
```diff
@@ -182,6 +182,10 @@ module.exports = {
         title: "Support",
         children: ["/content/sre/support/support.md"],
       },
+      {
+        title: "Multi-tenancy",
```
Comment (rasheedamir) [inline .vuepress/config.js]: @abdulhaseeb2 we should call this tenant-operator specifically as it's our product

File: content/sre/tenant-operator/overview.md [hunk 2]
```diff
@@ -2,10 +2,11 @@
 
 ## Overview
 
-It enables cluster administrators to host multiple tenants in a single Stakater Agility Platform. i.e.
+The idea of Tenant Operator is to use namespaces as independent sandboxes, where tenant applications can run independently from each other. To minimize cluster admin efforts, cluster admins shall configure Tenant Operator's custom resources, which then become a self-service system for tenants. Tenant Operator enables cluster administrators to host multiple tenants in a single Stakater Agility Platform, i.e.
```
Comment (rasheedamir) [inline content/sre/tenant-operator/overview.md]: Stakater App Agility Platform

File: content/sre/tenant-operator/overview.md [hunk 3]
```diff
@@ -56,37 +59,24 @@ spec:
...
 ```
 
-- Tenant CR has 3 kinds of users:
-  - `Owner`: Are users who will be owners of a tenant. They will have openshift admin-role assigned to their users and they can also create namespaces.
-  - `Edit`: Are user who will be editors of a tenant. They will have openshift edit-role assigned to their users.
-  - `View`: Are user who will be viewers of a tenant. They will have openshift view-role assigned to their users.
+* Tenant CR has 3 kinds of users:
+  + `Owner:` Users who will be owners of a tenant. They will have openshift admin-role assigned to their users and they can also create namespaces.
+  + `Edit:` Users who will be editors of a tenant. They will have openshift edit-role assigned to their users.
+  + `View:` Users who will be viewers of a tenant. They will have openshift view-role assigned to their users.
```
Comment (rasheedamir) [inline content/sre/tenant-operator/overview.md:58]: @abdulhaseeb2 we should clarify CRUDS; and instead list items

File: content/sre/tenant-operator/overview.md [hunk 4]
```diff
@@ -181,13 +189,45 @@ spec:
...
+kind: Namespace
+metadata:
+  labels:
+    stakater.com/tenant: development
+  name: build
+```
+
+* Namespace should have label `stakater.com/tenant` which contains the name of tenant to which it belongs to. The labels and annotationos specified in the operator config,  `ocp.labels.project` and `ocp.annotations.project` are inserted in the namespace by the controller.
+
+## Roles
```
Comment (rasheedamir) [inline content/sre/tenant-operator/overview.md]: @abdulhaseeb2 roles? seems to be misplaced; or maybe the heading is wrong

File: content/sre/tenant-operator/overview.md [hunk 5]
```diff
@@ -2,10 +2,11 @@
...
-It enables cluster administrators to host multiple tenants in a single Stakater Agility Platform. i.e.
+The idea of Tenant Operator is to use namespaces as independent sandboxes, where tenant applications can run independently from each other. To minimize cluster admin efforts, cluster admins shall configure Tenant Operator's custom resources, which then become a self-service system for tenants. Tenant Operator enables cluster administrators to host multiple tenants in a single Stakater Agility Platform, i.e.
 
-- enable to share managed applications to multiple tenants
-- enable to share Openshift cluster to multiple tenants
+* Share an **OpenShift cluster** with multiple tenants
+* Share **managed applications** with multiple tenants
+* Configure and manage tenants and their sandboxes
 
 **Why?**
```
Comment (rasheedamir) [inline content/sre/tenant-operator/overview.md]: @abdulhaseeb2 should we really have the **Why** section?

Comment (rasheedamir) [reply content/sre/tenant-operator/overview.md]: it seems to have a redundant information

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-70dedb79`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-0272de93`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-880aa3c8`

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-9e38577c`

**Review by rasheedamir — APPROVED** [review]

**Review by rasheedamir — CHANGES_REQUESTED** [review]
check my comments

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-225-de30f26d`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.165`

**Pull request merged**